pylint/checkers/spelling.py

```diff
@@ -44,6 +49,27 @@
 table = maketrans("", "")
 
 
+class WordsWithDigigtsFilter(Filter):
+    """
+    Skips words with digits
+    """
+
+    def _skip(self, word):
+        for char in word:
+            if char.isdigit():
```

Comment:
I timed it, its slower. Check the commit message where I did the change for the numbers.